.circleci/config.yml

```diff
@@ -379,9 +379,9 @@ workflows:
       - deploy-to-dev:
           requires:
             - update-dev-endpoints
-  #    - create-PR-to-master:
-  #        requires:
-  #          - deploy-to-dev
+      - create-PR-to-master:
+          requires:
+            - deploy-to-dev
```

Comment:
I remember this was failing because CircleCI does not have permissions to create a new PR.